tests/unit/app/endpoints/test_ols.py

```diff
@@ -193,12 +208,15 @@ def test_conversation_request(
 
 @patch("ols.src.query_helpers.question_validator.QuestionValidator.validate_question")
 @patch("ols.utils.config.conversation_cache.get")
+@patch("ols.utils.config.QuestionFilter.redact_question")
```

Comment:
same question here.....why do we need this?

Reply:
It does not need it . Fixed it.